libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -94,12 +93,24 @@ lua_scripts::script_info *lua_scripts::load_script(lua_State *L, char *filename)
 
 
     // find and create a sandbox for the new chunk
-    lua_getglobal(L, "get_sandbox_env");
-    if (lua_pcall(L, 0, LUA_MULTRET, 0)) {
-        gcs().send_text(MAV_SEVERITY_CRITICAL, "Scripting: Could not create sandbox: %s", lua_tostring(L, -1));
-        hal.console->printf("Lua: Scripting: Could not create sandbox: %s", lua_tostring(L, -1));
-        return nullptr;
-    }
+    lua_newtable(L);
+    luaopen_base_sandbox(L);
+    lua_pushstring(L, "math");
+    luaopen_math(L);
+    lua_settable(L, -3);
```

Comment:
what is -3 as an idx?

Reply:
Means 3 down from the top of the Lua stack. You can reference the stack with an absolute position from it's base if you know it (IE 0, 1, 2) or a virtual one, which is a negative number and is counted from the top of the Lua stack.